### DISCUSSION TITLE
JedisPool used to connect with RedisCluster

### DISCUSSION THREAD

**Discussion by shunmukh** [xetorthio/jedis#2909] [Q&A] [answered]
will there be a problem using jedispool to connect a redis cluster.as of now we have single shard in cluster and it works fine.
what happens if we specify less connections(8 is default) in connectional pool object in production ?

**Comment by sazzad16** [comment, marked as answer]
> what happens if we specify less connections

@shunmukh you would have less max connections to be operated at the same time. 

depending on your application and load, it may not be problem.

**Comment by shunmukh** [comment]
@sazzad16 is it ok to use jedispool instead of jediscluster ?

**Reply by sazzad16** [reply]
@shunmukh these are two different things. you choose one depending on your server type. you can't just use JedisCluster instead of JedisPool (for the same server).
